rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_reader :person, :book, :date
+
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    @person = person
+
+    book.rentals.push(self)
+    person.rentals.push(self)
+  end
+
+  def book=(book)
```

Comment:
- [ ] kindly remove the `=` after the name of the method. use ` def book(book)` instead of    `def book=(book)`